kotsadm/kustomize/overlays/dev/deployment.yaml

```diff
@@ -42,7 +54,9 @@ spec:
               value: http://replicated-app:3000
             - name: API_ADVERTISE_ENDPOINT
               value: http://127.0.0.1:30065
-            # - name: STORAGE_BASEURI
```

Comment:
this is handy for testing the ORAs code...

Reply:
oops